pipeline/data/dataset_importer.py

```diff
@@ -15,8 +15,8 @@
 import os
```

Comment:
I'm finding the naming of files a bit confusing. Do you have any ideas on how to be a bit less ambiguous?

We have `downloader` and `importer` which mean pretty much the same thing. In the tasks we have now `parallel` and `mono` when talking about datasets. Perhaps it would be nice to rename the files to match our new naming scheme and decide on `downloader` vs `importer` on terms. There's also some ambiguity around the fact that the parallel files are responsible for different things. `importers.py` downloads the parallel datasets, and the `dataset_importer.py` augments them.

Reply:
Agreed. I think we can call them `parallel_importer`, `mono_importer` and `paralle_downloaders` where `importer` is something that not only downloads data but does something else required to use the data, like shuffling or augmentation. The `downloader` just downloads the required amount of data from a resource.